index.js

```diff
@@ -11,6 +12,109 @@ function menuClickedHandler() {
+    image: 'images/SnapshootD2.jpg',
+    links: ['https://henarykhuraijam.github.io/MyPortfolio/', 'https://github.com/HenaryKhuraijam/MyPortfolio'],
+  },
+  {
+    id: 3,
+    name: 'Tonic',
+    summaries: { client: 'Self-Employed', role: 'Solo', year: '2022' },
+    details: `Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+     Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+     when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent`,
+    skills: ['html', 'css', 'javascripts'],
+    image: 'images/SnapshootD3.jpg',
+    links: ['https://henarykhuraijam.github.io/MyPortfolio/', 'https://github.com/HenaryKhuraijam/MyPortfolio'],
+  },
+  {
+    id: 4,
+    name: 'Multi-Post Stories',
+    summaries: { client: 'Self-Employed', role: 'Solo', year: '2022' },
+    details: `Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+     Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+     when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent`,
+    skills: ['html', 'css', 'javascripts'],
+    image: 'images/SnapshootD4.jpg',
+    links: ['https://henarykhuraijam.github.io/MyPortfolio/', 'https://github.com/HenaryKhuraijam/MyPortfolio'],
+  },
+];
+const showModal = (pid) => {
+  const curPro = pro[pid];
+  const sec = document.createElement('section');
+  sec.className = 'modal__card';
+  sec.id = 'modal-card';
+  sec.innerHTML = `<div class="cards" id="card1">
+  <div class="topic">
+  <h2 class="project-titles">${curPro.name} <img class="closebtn" id="closeButton" src="icons/cancelIconBlack.svg" alt="Cancel-Icon"></h2>
+    <ul class="inline-li project-summary">
+      <li class="client">${curPro.summaries.client}</li>
+      <li class="type">${curPro.summaries.role}</li>
+      <li class="year">${curPro.summaries.year}</li>
+    </ul> 
+  </div>
+  
+  <div class="snapshoot-placeholders modal">
+    <img class="snapshoot" src=${curPro.image} alt="">
+  
+  </div>
+  <div class="modal-discription">
+    
+    <p class="mobile-discription">
+    ${curPro.details} 
+    </p>
+    
+     <div class="language">
+    <ul class="inline-li">
+      <li class="tags">html</li>
+      <li class="tags">css</li>
+      <li class="tags">javascripts</li>
+      <li class="tags">Ruby</li>
+      <li class="tags">Bootstrap</li>
+  
+    </ul>
+   <div class="live-btn">
+    <button  class="btn" type=button"">See live <img class="live-icon" src="icons/liveIcon.svg" alt=""> </button>
+    <button  class="btn" type="button">See Source <img class="live-icon" src="icons/buttonIconGitHub.svg" alt=""></button></div>
+  </div>
+  </div>
+  </div>`;
```

Comment:
- [ ] Kindly work on the image responsiveness, it looks deformed on desktop, you can use a max width to make sure the modal does not exceed that width on desktop screens
<img width="1897" alt="image" src="https://user-images.githubusercontent.com/67957533/226126761-b3f8dcc0-e392-4003-a775-1dfc9784a1ae.png">

Reply:
Thank you finding out my mistakes